src/config.py

```diff
@@ -77,6 +77,7 @@
     (r'/api/build/?', 'biothings.web.handlers.MetadataSourceHandler'),
     (r'/api/status/?', 'biothings.web.handlers.StatusHandler'),
     (r'/api/suggestion/?', 'handlers.api.ValueSuggestionHandler'),
+    (r'/api/metakg/?', 'handlers.api.MetaKGHandler'),
```

Comment:
@sengineer0 we should also have a `/api/metakg/query` endpoint, just like `/api/query` is for querying SmartAPIDoc index.

Reply:
ok let's me add it

Reply:
@newgene It's updated.